lib/code-excerpt.js

```diff
@@ -1,14 +1,15 @@
-'use strict';
-const fs = require('fs');
-const equalLength = require('equal-length');
-const codeExcerpt = require('code-excerpt');
-const truncate = require('cli-truncate');
-const chalk = require('./chalk').get();
+import fs from 'fs';
```

Comment:
```suggestion
import fs from 'node:fs';
```

Any thoughts on switching to this while you're at it? I plan to enforce it in XO soon.

https://github.com/nodejs/node/issues/38343

Reply:
I'd be happy to let XO fix & enforce that when the time comes 😄